docs/source/ivr/ivr.rst

```diff
@@ -316,3 +316,8 @@ Custom exception
         }
 
     This will allow you to return an error code that your IVR provider can understand.
+
+Using IVR module with IVR providers
+===================================
+
+- :std:ref:`Using IVR with Verboice <using-ivr-with-verboice>`
```

Comment:
Don't put bare links as content of a subtopic. Add at least few words for a start